src/utils/symlinkPackageDependencies.js

```diff
@@ -127,12 +127,6 @@ export default async function symlinkPackageDependencies(
       actualBinFileRelative
     );
 
-    if (!pathIsInside(actualBinFile, project.pkg.nodeModules)) {
```

Comment:
Discussed with @jamiebuilds and we're removing this as it was for future plans around linking multiple bolt repos that didn't come to fruition.